docker/mysql/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:jessie
```

Comment:
"Updated 16 days ago". Seems like it's not sealed. But I can't see any other versions you could use ;(